encoder.go

```diff
@@ -47,9 +47,7 @@ func EncodeVarint(w io.Writer, i int64) (err error) {
 }
 
 func VarintSize(i int64) int {
-	var buf [10]byte
-	n := binary.PutVarint(buf[:], i)
-	return n
```

Comment:
great work. And learn new knowledge that right bit move can fill with 1.

Reply:
great work. And learn new knowledge that right bit move can fill with 1.